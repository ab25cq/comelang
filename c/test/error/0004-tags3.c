/*
PATTERN: tag
PATTERN: c:7:
*/

enum   X;
struct X;
