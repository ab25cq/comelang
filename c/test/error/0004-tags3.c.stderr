struct/union accessed by enum tag at test/error/0004-tags3.c:7:1
struct X;
^
